### ISSUE TITLE
(WCAG 2.1) Actionable control is not keyboard accessible

### ISSUE THREAD

**Issue by dkatzz** [civiform/civiform#5531]
OBSERVED RESULTS:

- The expandable/collapsible control "What are the eligibility requirements?" is not keyboard accessible and cannot be operated using keyboard navigation.
- ![Screenshot](https://github.com/civiform/civiform/assets/86739416/0489f5be-a697-4df6-a11e-59d16a63c6f3)

EXPECTED RESULTS:

- The expandable/collapsible control should be fully keyboard accessible, allowing users to expand and collapse its content using keyboard commands.

STEPS TO REPRODUCE:

- Navigate to [URL](https://demo.civiform.dev/)
- Navigate to and activate the ‘Apply’ for the Comprehensive sample program.
- Navigate to and activate “What is your address?”.
- Navigate to expandable/collapsible control "What are the eligibility requirements? using keyboard TAB or SHIFT TAB key.
- Observe the control is not keyboard accessible.

ENVIRONMENT DETAILS:

- Device(s) (brand, OS version, test key or release key): Windows 10 Enterprise (64-bit),MacOS Ventura Version 13.4
- Screen reader name and version: JAWS 2022.2211.7, NVDA 2022.3, VoiceOver
- Browser name and version: Chrome Version 113.0.5672.93

**Comment by melanie-exygy** [comment]
Completed in this [PR](https://github.com/civiform/civiform/pull/5463)
